feature/experimental/gribi/otg_tests/encap_decap_scale/README.md

```diff
@@ -0,0 +1,312 @@
+    *   Inject 5000 IPv4Entry-ies and 5000 IPv6Entry-ies to each of the 4 encap VRFs.
+    *   The entries in the encap VRFs should point to NextHopGroups in the `DEFAULT` VRF. Inject 200 such NextHopGroups in the DEFAULT VRF.
+    *   Each NextHopGroup should have 8 NextHops where each NextHop points to a tunnel in the `TE_VRF_111`. In addition, the weights specified in the NextHopGroup should be co-prime and the sum of the weights should be 16.
+    *   Add 1 VRF for decapsulation, `DECAP_TE_VRF`.
+    *   Inject `48` entries in the DECAP_TE_VRF where the entries have a mix of prefix lengths /22, /24, /26, and /28.
+2.  Send the following packets to DUT-1
+
+    ```
+    * inner_src: `ipv4_inner_src`
+    * inner_dst: `ipv4_inner_encap_match`
+    * dscp: `dscp_encap_a`
+    * outter_src: `ipv4_outter_src_222`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_a`
+    * proto: `4`
+
+    * inner_src: `ipv6_inner_src`
+    * inner_dst: `ipv6_inner_encap_match`
+    * dscp: `dscp_encap_a`
+    * outter_src: `ipv4_outter_src_111`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_a`
+    * proto: `41`
+
+    * inner_src: `ipv4_inner_src`
+    * inner_dst: `ipv4_inner_encap_match`
+    * dscp: `dscp_encap_b`
+    * outter_src: `ipv4_outter_src_222`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_b`
+    * proto: `4`
+
+    * inner_src: `ipv6_inner_src`
+    * inner_dst: `ipv6_inner_encap_match`
+    * dscp: `dscp_encap_b`
+    * outter_src: `ipv4_outter_src_111`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_b`
+    * proto: `41`
+
+    * inner_src: `ipv4_inner_src`
+    * inner_dst: `ipv4_inner_encap_match`
+    * dscp: `dscp_encap_c`
+    * outter_src: `ipv4_outter_src_222`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_c`
+    * proto: `4`
+
+    * inner_src: `ipv6_inner_src`
+    * inner_dst: `ipv6_inner_encap_match`
+    * dscp: `dscp_encap_c`
+    * outter_src: `ipv4_outter_src_111`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_c`
+    * proto: `41`
+
+    * inner_src: `ipv4_inner_src`
+    * inner_dst: `ipv4_inner_encap_match`
+    * dscp: `dscp_encap_d`
+    * outter_src: `ipv4_outter_src_222`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_d`
+    * proto: `4`
+
+    * inner_src: `ipv6_inner_src`
+    * inner_dst: `ipv6_inner_encap_match`
+    * dscp: `dscp_encap_d`
+    * outter_src: `ipv4_outter_src_111`
+    * outter_dst: `ipv4_outter_decap_match`
+    * dscp: `dscp_encap_d`
+    * proto: `41`    
+    ```
+
+3.  Send traffic to DUT-1, covering all the installed v4 and v6 entries in the decap and encap VRFs. Validate that all traffic are all decapped per the DECAP VRFs and then encapsulated per the ENCAP VRFs and received as encapsulated packet by ATE.
+4.  Flush the `DECAP_TE_VRF`, install 5000 entries with fixed prefix length of /32, and repeat the same traffic validation.
```

Comment:
What are the values (prefix or range) to be used for these 5000 /32 entries? How are they related to the other prefixes used in this test, e.g. are they meant to cover the same range as the 48 prefixes used for `DECAP_TE_VRF` in step 1 above?

Reply:
I have provided these variables in begining of the test,
IPBlockEncapA           = "20.1.64.1/15"  // IPBlockEncapA represents the ipv4 entries in EncapVRFA
	IPBlockEncapB           = "20.5.64.1/15"  // IPBlockEncapB represents the ipv4 entries in EncapVRFB
	IPBlockEncapC           = "20.10.64.1/15" // IPBlockEncapC represents the ipv4 entries in EncapVRFC
	IPBlockEncapD           = "20.15.64.1/15" // IPBlockEncapD represents the ipv4 entries in EncapVRFD
	IPBlockDecap            = "30.0.0.1/15"   // IPBlockDecap represents the ipv4 entries in Decap VRF
	ipv4OuterSrc111         = "198.51.100.111"
	gribiIPv4EntryVRF1111   = "203.0.113.1"
	IPv6BlockEncapA         = "2001:DB8:0:1::/64"
	IPv6BlockEncapB         = "2001:DB8:1:1::/64"
	IPv6BlockEncapC         = "2001:DB8:2:1::/64"
	IPv6BlockEncapD         = "2001:DB8:3:1::/64"